Core/src/main/java/com/plotsquared/core/command/ListCmd.java

```diff
@@ -123,10 +123,7 @@ private String[] getArgumentList(PlotPlayer<?> player) {
     }
 
     public void noArgs(PlotPlayer<?> player) {
-        player.sendMessage(
-                TranslatableCaption.of("commandconfig.subcommand_set_options_header"),
-                Templates.of("values", Arrays.toString(getArgumentList(player)))
-        );
+        onCommand(player, new String[]{"mine"});
```

Comment:
Feels kinda hacky and dirty imo - The logic for the `mine`-subcmd should be extracted into a (private) method and called by that. The noArgs method has no real use-case anymore.